code/modules/food_and_drinks/machinery/microwave.dm

```diff
@@ -466,20 +477,36 @@
 	. = ..()
 	if((machine_stat & NOPOWER) && operating)
 		pre_fail()
-		eject()
+		eject(force = TRUE) // monkestation edit: microwave "enhancements"
```

Comment:
not proper modularization; see the modularization doc

Reply:
yeah I don't know how I'm supposed to modularize this